plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF

```diff
@@ -7,3 +7,6 @@ Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
+Require-Bundle: org.eclipse.equinox.common
```

Comment:
Unless it causes problems, I think you should use Import-Package for this instead: http://stackoverflow.com/questions/1865819/when-should-i-use-import-package-and-when-should-i-use-require-bundle